tests/unit/test_utils.py

```diff
@@ -207,6 +209,135 @@ def test_fail(self, x: torch.Tensor):
+                        " of",
+                        " the",
+                        " circ",
+                    ],
+                ),
+                call("Tokenized answer:", ["le", "."]),
+                call("Top 0th token. Logit: 16.69 Prob: 38.00% Token: |a|"),
+                call("Top 1th token. Logit: 15.71 Prob: 14.30% Token: |let|"),
+                call("Top 2th token. Logit: 15.48 Prob: 11.37% Token: |ump|"),
+                call("Top 3th token. Logit: 14.94 Prob:  6.64% Token: |u|"),
+                call("Top 4th token. Logit: 13.96 Prob:  2.49% Token: |lets|"),
+                call("Top 5th token. Logit: 13.23 Prob:  1.19% Token: |ut|"),
+                call("Top 6th token. Logit: 13.17 Prob:  1.13% Token: |umn|"),
+                call("Top 7th token. Logit: 13.01 Prob:  0.96% Token: |us|"),
+                call("Top 8th token. Logit: 12.96 Prob:  0.91% Token: | is|"),
+                call("Top 9th token. Logit: 12.89 Prob:  0.85% Token: |umb|"),
+                call("Top 0th token. Logit: 15.30 Prob: 44.91% Token: |th|"),
+                call("Top 1th token. Logit: 12.67 Prob:  3.22% Token: | 1|"),
+                call("Top 2th token. Logit: 12.46 Prob:  2.60% Token: |.|"),
+                call("Top 3th token. Logit: 12.33 Prob:  2.30% Token: | 2|"),
+                call("Top 4th token. Logit: 11.98 Prob:  1.62% Token: |,|"),
+                call("Top 5th token. Logit: 11.85 Prob:  1.41% Token: |-|"),
+                call("Top 6th token. Logit: 11.79 Prob:  1.33% Token: | and|"),
+                call("Top 7th token. Logit: 11.62 Prob:  1.13% Token: | 3|"),
+                call("Top 8th token. Logit: 11.56 Prob:  1.06% Token: |thal|"),
+                call("Top 9th token. Logit: 11.48 Prob:  0.98% Token: |an|"),
+            ],
+            [
+                call(
+                    "Performance on answer token:\n[b]Rank: 93       Logit:  9.99 Prob:  0.05% Token: |le|[/b]"
+                ),
+                call(
+                    "Performance on answer token:\n[b]Rank: 2        Logit: 12.46 Prob:  2.60% Token: |.|[/b]"
+                ),
+                call("[b]Ranks of the answer tokens:[/b] [('le', 93), ('.', 2)]"),
+            ],
+        ),
+    ],
+)
+@patch("builtins.print")
+@patch("transformer_lens.utils.rprint")
+def test_test_prompt(
+    mocked_rprint,
+    mocked_print,
+    prepend_space_to_answer,
+    expected_print_calls,
+    expected_rprint_calls,
+):
+    utils.test_prompt(
+        "The circumference is the perimeter of the circ",
+        "le.",
+        model,
+        prepend_space_to_answer=prepend_space_to_answer,
+    )
+
+    for received_call, expected_call in zip(
+        mocked_print.mock_calls, expected_print_calls
+    ):
+        assert received_call == expected_call
+    for received_call, expected_call in zip(
+        mocked_rprint.mock_calls, expected_rprint_calls
+    ):
+        assert received_call == expected_call
```

Comment:
I don't think this is needed (and it's best practice to just test one thing/feature in a test - in this case tokenisation).

```suggestion
```